File: app/src/main/java/com/bitvale/telegramcharts/Prefs.java
package com.bitvale.telegramcharts;

import android.content.Context;
import android.content.SharedPreferences;

/**
 * Created by Alexander Kolpakov (jquickapp@gmail.com) on 17-Mar-19
 */
class Prefs {

    private static final String PREFS_FILENAME = "com.bitvale.telegramcharts.prefs";
    private static final String THEME = "theme";

    private SharedPreferences prefs;

    Prefs(Context context) {
        prefs  = context.getSharedPreferences(PREFS_FILENAME, 0);
    }

    int getTheme() {
        return prefs.getInt(THEME, R.style.AppThemeLight);
    }

    void setTheme(int theme) {
        prefs.edit().putInt(THEME, theme).apply();
    }
}

File: app/src/main/java/com/bitvale/telegramcharts/Prefs.kt
package com.bitvale.telegramcharts

import android.content.Context
import android.content.SharedPreferences

/**
 * Created by Alexander Kolpakov (jquickapp@gmail.com) on 12-Mar-19
 */
class Prefs(context: Context) {
    val PREFS_FILENAME = "com.bitvale.telegramcharts.prefs"
    val THEME = "theme"

    val prefs: SharedPreferences = context.getSharedPreferences(PREFS_FILENAME, 0);

    var theme: Int
        get() = prefs.getInt(THEME, R.style.AppThemeLight)
        set(value) = prefs.edit().putInt(THEME, value).apply()
}